pages/linux/mmcli.md

```diff
@@ -0,0 +1,12 @@
+# mmcli
+
+> Control and monitor the ModemManager.
+> More information: <https://www.freedesktop.org/software/ModemManager/man/1.0.0/mmcli.8.html>.
```

Comment:
Refer to `latest` instead of `1.0.0` ?
```suggestion
> More information: <https://www.freedesktop.org/software/ModemManager/man/latest/mmcli.1.html>.
```

Reply:
I tried using "latest", but I didn't think to chanege the section number like that—smart!